frontend/src/lib/util.ts

```diff
@@ -183,8 +183,11 @@ export function useFilterFunc<
  * @param returnWhenNoClusters return this value when no clusters are found.
  * @returns the cluster group from the URL.
  */
-export function getSelectedClusters(returnWhenNoClusters: string[] = []): string[] {
-  const clusterFromURL = getClusterPathParam();
+export function getSelectedClusters(
+  returnWhenNoClusters: string[] = [],
+  urlPath?: string
```

Comment:
This new param should also be documented.

Reply:
done